src/encoded/schemas/supplementary_file.json

```diff
@@ -1,5 +1,6 @@
 {
     "title": "Supplementary File",
+    "description": "Submitted files not generated by DAC",
```

Comment:
I'm not sure if this description is accurate, since we have DAC-generated files that will be submitted as supplementary files (the downsampled HG005 files from Alice Lee's lab) and not all non-DAC-generated submitted files are considered supplementary files. Maybe we can say that supplementary files are files that do not fall into other file item types in the schema, namely UnalignedReads, AlignedReads, and VariantCalls? I'm not sure if this is technically true either, though; there's probably a better description for SupplementaryFiles than this.

Reply:
That's a good point. Maybe SupplementaryFiles are "not generated by the DAC analysis pipeline"? Or should I just say that they are files that don't fit into any of the other file types?

Reply:
I think what you changed the description to is more accurate! I am not sure if it's 100% accurate still, since we generally don't process any of the TTD files as well, but it should be alright for now.